testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py

```diff
@@ -0,0 +1,598 @@
+
+# Regions
+Delaware_Region = {"include":[np.array([-77, -69.8 ,35.5, 41])],
+                   "exclude":[]}
+
+# Coastlines
+US_East_Coast = {"include":[np.array([-81.7,-62.3,25.1,46.24])],
+                 "exclude":[np.array([-66.0,-64.0,31.5,33.0]),    # Bermuda
+                             np.array([-79.75,-70.0,20.0,28.5])]} # Bahamas
+US_Gulf_Coast = {"include":[np.array([-98.0,-80.0,24.0,31.0]),     
+                            np.array([-91.0,-86.0,20.0,22.0])     # Yucatan
+                           ],   
+                 "exclude":[]}
+
+US_West_Coast = {"include":[np.array([-127.0,-116.0,32.5,49.0]),
+                            np.array([-117.5,-109.0,23.0,32.5])   # Baja
+                            ],
+                 "exclude":[np.array([-116.5,-115.0,32.8,33.8]),  # Salton Sea
+                            np.array([-120.5,-116.5,35.5,40.5])]} # Lake Tahoe, etc.
+Alaska        = {"include":[np.array([-170.0,-141.0,49.0,72.0]),
+                            np.array([-141.0,-129.5,49.0,61.0]),  # Southeast
+                            np.array([-129.5,-121.0,49.0,55.0])   # Connects AK to CA
+                           ],
+                 "exclude":[]}
+
+# Combined coastlines
+CONUS = {"include":[],"exclude":[]}
+CONUS["include"].extend(US_East_Coast["include"]) 
+CONUS["include"].extend(US_Gulf_Coast["include"]) 
+CONUS["include"].extend(US_West_Coast["include"])
+CONUS["exclude"].extend(US_East_Coast["exclude"])
+CONUS["exclude"].extend(US_West_Coast["exclude"])
+
+Continental_US = {"include":[],"exclude":[]}
+Continental_US["include"].extend(CONUS["include"])
+Continental_US["include"].extend(Alaska["include"])
+Continental_US["exclude"].extend(CONUS["exclude"])
+
+#----------------
+# Plotting boxes
+#----------------
+
+Western_Atlantic = np.array([-98.186645, -59.832744, 7.791301 ,45.942453])
+Contiguous_US = np.array([-132.0,-59.832744,7.791301,51.0])
+North_America = np.array([-175.0,-60.0,7.5,72.0])
+Delaware = np.array([-77, -69.8 ,35.5, 41])
+Entire_Globe = np.array([-180,180,-90,90])
+
+#-----------------
+# Restrict Boxes
+#-----------------
+
+Delaware_restrict = {"include":[np.array([[-75.853,39.732],
+                                          [-74.939,36.678],
+                                          [-71.519,40.156],
+                                          [-75.153,40.077]]),
+                                np.array([[-76.024,37.188],
+                                          [-75.214,36.756],
+                                          [-74.512,37.925],
+                                          [-75.274,38.318]])],
+                     "exclude":[]}
+
+######################################################################################
+# User-defined inputs
+######################################################################################
+
+params = {
+
+# Path to bathymetry data and name of file
+"nc_file": "/users/sbrus/climate/bathy_data/SRTM15_plus/earth_relief_15s.nc",
```

Comment:
I agree. I'll change the name to default_params, since each case's define_base_mesh.py imports and then modifies each option.

Can I add the bathy data file to the initial_condition_database so it can be linked with the config_base_mesh.xml file?